src/modules/launcher/Plugins/Microsoft.Plugin.Program/Programs/IAppxManifestApplication.cs

```diff
@@ -4,7 +4,7 @@
 
 using System;
 using System.Runtime.InteropServices;
-using static Microsoft.Plugin.Program.Programs.UWP;
+using HRESULT = Wox.Plugin.Common.Win32.HRESULT;
```

Comment:
Feedback from earlier, this shouldn't be like this. namespace / enum collision

Reply:
@cyberrex5 probably missed this one? as I see you changed others..

Reply:
oh yeah i missed it